CHANGELOG.md

```diff
@@ -8,12 +8,15 @@
 ### Fixed
 * Allow backlinks between files. ([#1015](https://github.com/realm/realm-dart/issues/1015))
 * Fix issue with accessing properties after traversing a backlink. ([#1018](https://github.com/realm/realm-dart/issues/1018))
+* Fix a race condition which could result in "operation cancelled" errors being delivered to async open callbacks rather than the actual sync error which caused things to fail (Core upgrade, since the introduction of async open).
+* Bootstraps will not be applied in a single write transaction - they will be applied 1MB of changesets at a time, or as configured by the SDK (Core upgrade, since v12.0.0).
+* Fix database corruption and encryption issues on apple platforms, reported in several bugs listed in the PR. (Core upgrade, since v11.8.0)
```

Comment:
Need to reword this or include a link to the PR. Also, need to remove "since ...".

Reply:
What do you prefer? Last time I as asked to remove the PR links

Reply:
Generally, we don't include PR links, but the wording here mentions "issues linked in the PR". We should either explain what the issues are or link to the PR to allow people to read up themselves.

Reply:
I have pushed a suggestion